src/matrixrtc/EncryptionManager.ts

```diff
@@ -146,9 +149,9 @@ export class EncryptionManager implements IEncryptionManager {
         this.manageMediaKeys = false;
         this.joined = false;
     }
+
     // TODO deduplicate this method. It also is in MatrixRTCSession.
-    private isMyMembership = (m: CallMembership): boolean =>
-        m.sender === this.client.getUserId() && m.deviceId === this.client.getDeviceId();
+    private isMyMembership = (m: CallMembership): boolean => m.sender === this.userId && m.deviceId === this.deviceId;
```

Comment:
While we are at it. Do you think it makes sense to move this into types.ts and deduplicate it? (as the comment says?)